packages/rest/src/lib/handlers/SequentialHandler.ts

```diff
@@ -190,8 +283,10 @@ export class SequentialHandler {
 					`  Retry After    : ${retryAfter}ms`,
 				].join('\n'),
 			);
-			// Wait the retryAfter amount of time before retrying the request
-			await sleep(retryAfter);
+			// If caused by a sublimit, wait it out here so other requests on the route can be handled
+			if (sublimitTimeout) {
+				await sleep(sublimitTimeout);
```

Comment:
Pretty sure this will block other requests, no?

Reply:
You are right, gonna need to think on this one since the handling was slightly different and I missed that.